paper.md

```diff
@@ -0,0 +1,91 @@
+
+# Statement of need
+
+Deep learning is increasingly used in medical image processing [@kim_deep_2019]. It provides automated solutions to repetitive and/or tedious tasks such as the segmentation of pathological structures. However, medical imaging data present many challenges: datasets are often not publicly-available, ground truth labels are scarce due to the limited availability of expert raters, and needs can be very specific and tailored to particular datasets (e.g., segmentation of spinal tumors on sagittal MRI T2-weighted scans). Thus, offering solution for convenient training models (or fine-tuning of pre-existing models) is needed.
+
+We present `ivadomed`, a deep learning toolbox dedicated to medical data processing. `ivadomed` aims to support the integration of deep learning models into the clinical routine, as well as state-of-the-art academic biomedical research. It features intuitive command-line tools for end-to-end training and evaluation of various deep learning models. The package also includes pre-trained models that can be used to accommodate specific datasets with transfer learning.
+
+Another challenge of medical imaging is the heterogeneity of the data across clinical centers, in terms of image features (e.g., contrast, resolution) and population demographics. This makes it challenging to create models that can generalize well across the many existing datasets. Recent cutting-edge methods address this problem, such as FiLM [@perez2017film] and HeMis [@havaei2016hemis], however they are usually not implemented within a comprehensive framework that enables end-to-end training and experimentation. In addition to providing these advanced architectures, `ivadomed` features multiple uncertainty estimation methods (aleatoric and epistemic), losses adapted to imbalanced classes and non-binary predictions. Each step can be conveniently customized via a single configuration file, and at the same time, the code is highly modular to allow addition/modification of architecture or pre-/post-processing steps.
+
+# Software description
+
+`ivadomed` is based on PyTorch framework [@paszke2017automatic] with GPU acceleration supported by CUDA. It can easily be installed via [PyPI](https://pypi.org/project/ivadomed/) and the whole package is tested with a [continuous integration](https://github.com/ivadomed/ivadomed/actions?query=workflow\%3A\%22Run+tests\%22) framework. The project website, which includes user and API documentation, is available at https://ivadomed.org. The name `ivadomed` is a portmanteau between [*IVADO (The Institute for data valorization)*](https://ivado.ca) and *medical*.
+
```

Comment:
I would remove because: 
- this section is optional
- the info is "somewhat" generic and can easily be found on the website
- we need to space

```suggestion
```